core/src/rpc.rs

```diff
@@ -3029,6 +3042,9 @@ pub mod tests {
         let cluster_info = Arc::new(ClusterInfo::default());
         let tpu_address = cluster_info.my_contact_info().tpu;
         let bank_forks = new_bank_forks().0;
+        let (sender, receiver) = channel();
```

Comment:
If this looks too ugly, we can follow TPU precedent and have JsonRpcRequestProcessor's constructor create the channel and return the `Receiver<TransactionInfo>` and feed it right into the `SendTransactionService` constructor.

Reply:
fixed